### ISSUE TITLE
virtual file mapping error

### ISSUE THREAD

**Issue by Pumpumpwirum** [Pumpumpwirum/tomita-parser#1]
Can't find a problem. Everything in the files should be correct as it is copied from the examples. The route is also correct.

[tom.zip](https://github.com/Pumpumpwirum/tomita-parser/files/1753069/tom.zip)

**Comment by vbocharov** [comment]
Думаю, что config.proto и остальные файлы лежат в папке tom. Если это так, то зайдите в неё и запускайте парсер оттуда. В данный момент он ищет config.proto в текущей папке (...\Marina\) и не находит.

**Comment by vbocharov** [comment]
Лучше заводить тикеты прямо там ( https://github.com/yandex/tomita-parser/issues ), чтобы набиралась база вопросов и ответов.

**Comment by kak-to-tak** [comment]
Спасибо большое!

27 фев 2018 г. 0:07 пользователь "Victor Bocharov" <notifications@github.com>
написал:

**Comment by vbocharov** [comment]
Заработало?

**Comment by kak-to-tak** [comment]
Да. Извините, пожалуйста, за беспокойство. Безумно неудобно из-за такой
глупой ошибки со своей стороны.

27 фев 2018 г. 0:41 пользователь "Victor Bocharov" <notifications@github.com>
написал:

**Comment by vbocharov** [comment]
Всё ок.
